### PR TITLE
use tsdoc

### LINKED ISSUE #47: Change JSDoc to TSDoc
## Summary

What: Convert JSDoc blocks (though we use a mix of JSDoc and Deno's standard), to TSDoc blocks in Typescript files

Why: We primarily use TypeScript so it makes sense to use TSDoc. Types are already documented in the method/property anyways.

## Acceptance Criteria

- [x] All doc blocks converts to TSDoc

## Example Pseudo Code (for implementation)

```
* @param string filename
*        The filename to do something with

* @param filename - The filename to do something with
```

## Resources

TSDoc is very similar, there is really only a few minor changes to make to each doc block

https://github.com/microsoft/tsdoc

### PR DESCRIPTION
**Pull request by crookse**
Closes #47 

**Description**

* Converts all docblocks from JSDoc to TSDoc
* Improves semantics in some parts of the code

### CODE REVIEW

File: src/event_emitter.ts [hunk 1]
```diff
@@ -79,78 +77,71 @@ export class EventEmitter {
...
+  public getChannel(channelName: string): Channel {
+    return this.channels[channelName];
   }
 
   /**
-   * @return any
-   *     Return all channels.
+   * Get all of the channels.
+   *
+   * @return An array with all of the channel names.
```
Comment (ebebbington) [inline src/event_emitter.ts:125]: `return` to `returns`

File: src/event_emitter.ts [hunk 2]
```diff
@@ -204,12 +191,8 @@ export class EventEmitter {
    * @description
    *     Send a message to a channel, excluding the sender.
    *
```
Comment (ebebbington) [inline src/event_emitter.ts:193]: `@description` is left in

File: src/event_emitter.ts (hunk 1 above)
Comment (crookse) [reply src/event_emitter.ts]: fixed: d8fb3b3

File: src/event_emitter.ts (hunk 2 above)
Comment (crookse) [reply src/event_emitter.ts]: fixed: https://github.com/drashland/sockets/pull/51/commits/3bad69b22ceb5f70569bf53c73cfc63738ea3028

**Pull request merged**